src/code/client.coffee

```diff
@@ -229,6 +230,12 @@ class CloudFileManagerClient
             return @alert(err) if err
             @_fileOpened content, metadata, {openedContent: content.clone()}, @_getHashParams metadata
 
+  openUrlFile: (url) ->
+    urlProvider = new URLProvider()
```

Comment:
I'd keep it as is but maybe add a comment with why it isn't instantiated like the rest of them.